### PR TITLE
MSVC 14.38.33135

### PR DESCRIPTION
**Pull request by mgovers**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* ~[ ] Reset the build number to `0` (if the version changed)~ not applicable
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Relates to #73 (but does not bump because 14.39 is not deployed everywhere yet)

### CODE REVIEW

**Comment by mgovers** [comment]
@conda-forge-admin, please rerender

**Comment by mgovers** [comment]
@conda-forge-admin, please rerender

**Comment by mgovers** [comment]
@conda-forge-admin, please rerender

**Comment by mgovers** [comment]
only one combination failed for [c7e3029](https://github.com/conda-forge/vc-feedstock/pull/76/commits/c7e3029d0f802a00aaa08af029f0a4433dc7d2be): https://github.com/conda-forge/vc-feedstock/pull/76/checks?check_run_id=25090610225

Trying out a couple combinations to try to find the correct one.

**Comment by mgovers** [comment]
I'm assuming that by "Ensured license file being packaged", you mean the `index` file in the artifact:

image

I checked all 8 artifacts and all of them contain such index file (although the contents differ)

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -16,35 +16,35 @@ vsver:
...
  - 14.29.30139
  - 14.16.27033
 # the VS update version.  This is the middle digit in the version
 # reported in the VS help->about UI.  It is perhaps a more readily
 # referenceable number.
 update_version:
- - 8
- - 8
+ - 9
+ - 9
```
Comment (isuruf) [inline recipe/conda_build_config.yaml:28]: This should have been `8`

Comment (isuruf) [reply recipe/conda_build_config.yaml:28]: Sending new PR

**Pull request merged**